decidim-admin/lib/decidim/admin/test/admin_participatory_space_access_examples.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+shared_examples "showing the page" do
```

Comment:
This example is too generic. "showing the page" is an example that could refer to : 
- Page component
- Static pages 
- Any particular page. 
- On a later stage, we may think this example as something trivial that answers to "is the current page loaded?"

Reply:
Yes, it's a lousy name. I've renamed it to ['accessing the participatory space'](https://github.com/decidim/decidim/pull/10968/commits/829f3a7138512129ab86198e7459e5c20255c365)